allensdk/brain_observatory/ecephys/__init__.py

```diff
@@ -0,0 +1,18 @@
+from typing import NamedTuple
+
+import numpy as np
+
+
+class RunningSpeed(NamedTuple):
```

Comment:
This is a class that can be promote up and out of ecephys, so that other modules can use the same dataclass